docs/basics/101-127-yoda.rst

```diff
@@ -234,7 +234,7 @@ compute outputs, and the code and outputs are captured,
 version-controlled, and linked to the input data. Each input data in turn
 is a (potentially nested) subdataset, but this is not visible
 in the directory hierarchy.
-Figure 3 in comparison emphasizes a process view on a project and
+Figure  in comparison emphasizes a process view on a project and
```

Comment:
Here, I'd keep the (corrected) figure number, too
```suggestion
Figure 6.2, in comparison, emphasizes a process view on a project and
```

Reply:
Hello @mslw, thanks for adding. Of course I approve.